unix_integration/src/unix_proto.rs

```diff
@@ -16,6 +17,59 @@ pub struct NssGroup {
     pub members: Vec<String>,
 }
 
+#[derive(Serialize, Deserialize, Debug, Default)]
+pub enum PamMessageStyle {
+    #[default]
+    PamPromptEchoOff,
+    PamPromptEchoOn,
+    PamErrorMsg,
+    PamTextInfo,
+}
+
+impl PamMessageStyle {
+    pub fn value(&self) -> i32 {
```

Comment:
This ugly mess is to allow the PamMessageStyle enum to map correctly to a c_int in pam (i32). Perhaps this won't work everywhere?